packages/terra-functional-testing/tests/jest/services/wdio-selenium-docker-service.test.js

```diff
@@ -149,7 +149,7 @@ describe('WDIO Selenium Docker Service', () => {
 
       const composeFilePath = path.resolve(__dirname, '../../../src/docker/docker-compose.yml');
 
-      expect(mockExec).toHaveBeenCalledWith(`TERRA_SELENIUM_DOCKER_VERSION=3.14.0-helium docker stack deploy -c ${composeFilePath} wdio`);
+      expect(mockExec).toHaveBeenCalledWith(`docker stack deploy -c ${composeFilePath} wdio`);
     });
```

Comment:
There is an exiting test below that verifies the specified version of selenium is invoked if provided:

https://github.com/cerner/terra-toolkit/blob/26aca0a25b7156ff3e127ad0ac05a720b83dee7f/packages/terra-functional-testing/tests/jest/services/wdio-selenium-docker-service.test.js#L155-L169